tests/basic.rs

```diff
@@ -7,6 +7,18 @@ fn new() {
     let _map = FlurryHashMap::<usize, usize>::new();
 }
 
+#[test]
+fn debug_format() {
+    let map = FlurryHashMap::<usize, usize>::new();
+    {
+        let g = epoch::pin();
+        map.insert(1, 0, &g);
+        map.insert(2, 1, &g);
+        map.insert(3, 2, &g);
+        map.insert(4, 3, &g);
+    }
+    assert_eq!("{3: 2, 1: 0, 2: 1, 4: 3}", format!("{:?}", map));
```

Comment:
Hmm, this assertion will probably fail since iteration order is random..

Reply:
Yep, already removed it